redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -131,10 +131,10 @@ internal open class ViewLazyList private constructor(
   override fun margin(margin: Margin) {
     with(density) {
       recyclerView.updatePaddingRelative(
-        start = margin.start.toPx().toInt(),
-        top = margin.top.toPx().toInt(),
-        end = margin.end.toPx().toInt(),
-        bottom = margin.bottom.toPx().toInt(),
+        start = margin.start.toPxInt(),
+        top = margin.top.toPxInt(),
+        end = margin.end.toPxInt(),
+        bottom = margin.bottom.toPxInt(),
```

Comment:
Looks like this went from truncating to rounding and changed the snapshot tests in a non-trivial way. Needs recaptured.

Reply:
Yep, that’s it. Good catch